tools/cli/adminClusterCommands.go

```diff
@@ -87,27 +87,43 @@ func AdminRebalanceStart(c *cli.Context) {
 	tcCtx, cancel := newContext(c)
 	defer cancel()
 
-	options := cclient.StartWorkflowOptions{
-		ID:                           failovermanager.RebalanceWorkflowID,
-		WorkflowIDReusePolicy:        cclient.WorkflowIDReusePolicyAllowDuplicate,
-		TaskList:                     failovermanager.TaskListName,
-		ExecutionStartToCloseTimeout: time.Minute,
-		Memo: map[string]interface{}{
-			common.MemoKeyForOperator: getOperator(),
-		},
-	}
-
+	workflowId := failovermanager.RebalanceWorkflowID
 	rbParams := &failovermanager.RebalanceParams{
 		BatchFailoverSize:              100,
 		BatchFailoverWaitTimeInSeconds: 10,
 	}
-	wf, err := client.StartWorkflow(tcCtx, options, failovermanager.RebalanceWorkflowTypeName, rbParams)
+	input, err := json.Marshal(rbParams)
+	if err != nil {
+		ErrorAndExit("Failed to serialize params for failover workflow", err)
+	}
+	request := &types.StartWorkflowExecutionRequest{
+		Domain:                              common.SystemLocalDomainName,
+		WorkflowID:                          failovermanager.RebalanceWorkflowID,
+		RequestID:                           uuid.New(),
+		Identity:                            getWorkerIdentity(""),
+		WorkflowIDReusePolicy:               types.WorkflowIDReusePolicyAllowDuplicate.Ptr(),
+		ExecutionStartToCloseTimeoutSeconds: common.Int32Ptr(60),
+		Input:                               input,
+		TaskList: &types.TaskList{
+			Name: failovermanager.TaskListName,
+		},
+		Memo: &types.Memo{
+			Fields: map[string][]byte{
+				common.MemoKeyForOperator: []byte(getOperator()),
```

Comment:
This will likely need conversion to json as well. https://github.com/uber-go/cadence-client/blob/master/internal/internal_workflow_client.go#L1165